release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1726,6 +1726,43 @@ In the following tables, features are marked with the following statuses:
 
 * A regression in the behaviour of `libreswan` caused some nodes with IPsec enabled to lose communication with pods on other nodes in the same cluster. To resolve this issue, consider disabling IPsec for your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-43713[*OCPBUGS-43713*])
 
+* There is a known issue in {product-title} {product-version} which causes the cluster's masquerade subnet to be set to `169.254.169.0/29` if the `ovnkube-node` daemon set is deleted. When the masquerade subnet is set to `169.254.169.0/29`, `UserDefinedNetwork` custom resources (CRs) cannot be created.
++
+[NOTE]
+====
+* If your masquerade subnet has been configured at Day 2 by making changes to the `network.operator` CR, it will not be reverted to `169.254.169.0/29`.
+* If a cluster has been upgraded from {product-title} 4.16, the masquerade subnet remains `169.254.169.0/29` for backward compatibility. The masquerade subnet should be changed to a subnet with more IPs, for example, `169.254.0.0/17`, to use the user-defined networks feature.
+====
++
+This known issue occurs after performing one of the following actions:
++
+[cols="2", options="header"]
+|===
+| Action | Consequence
+
+| You have restarted the OVN-Kubernetes daemon set.
+| The masquerade subnet is reverted to `169.254.169.0/29`, which does not support `UserDefinedNetwork` CRs.
+
+| You have deleted the OVN-Kubernetes daemon set.
+| The masquerade subnet is reverted to `169.254.169.0/29`, which does not support `UserDefinedNetwork` CRs. Additionally, OVN-Kubernetes pods crash and remain in a `CrashLoopBackOff` state.
+|===
++
+As a temporary workaround, you can delete the `UserDefinedNetwork` CR and then restart all `ovnkube-node` pods by running the following command:
++
+[source,terminal]
+----
+$ oc delete ovnkube-node -n openshift-ovn-kubernetes 
+----
++
+The `ovnkube-node` pods automatically restart and the masquerade subnet is set to `169.254.0.0/17` for IPv4. As a result, `NetworkAttachmentDefinition` or `UserDefinedNetwork` CRs can be created.
++
+[IMPORTANT]
+====
+Do not delete the OVN-Kubernetes daemon set when deleting `ovnkube-node` pods. This reverts the masquerade subnet to `169.254.169.0/29`.
+====
++
+This issue will be fixed in a future version of {product-title}. (link:https://issues.redhat.com/browse/OCPBUGS-49662[*OCPBUGS-49662*])
```

Comment:
Wondering about the inclusion of this sentence per https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future.

Reply:
I always felt like "in a future version" was vague enough to avoid that rule, but maybe I should just remove it altogether to be safe.